src/scenes/components/Footer/Footer.js

```diff
@@ -1,15 +1,150 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import styled from 'styled-components'
 
+import { H2, Body2 } from '../../../components/typography'
+import facebookIcon from '../../../assets/img/icon-facebook.svg'
+import instagramIcon from '../../../assets/img/icon-instagram.svg'
+import linkedinIcon from '../../../assets/img/icon-linkedin.svg'
+import twitterIcon from '../../../assets/img/icon-twitter.svg'
+import youtubeIcon from '../../../assets/img/icon-youtube.svg'
+
+const socialIcons = {
+  facebook: facebookIcon,
+  instagram: instagramIcon,
+  linkedin: linkedinIcon,
+  twitter: twitterIcon,
+  youtube: youtubeIcon,
+}
+
 const Container = styled.section`
-  border: 1px solid;
   flex-shrink: 0;
+  width: 100%;
+  padding: 84px 0 80px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: flex-start;
+  background-color: ${props => props.theme.color('neutral', 'base')};
+  color: ${props => props.theme.color('neutral', 'bright')};
+`
+
+const WebConferencesContainer = styled.div`
+  text-align: center;
+`
+
+const SocialContainer = styled.div`
+  text-align: left;
+`
+
+const WebConferencesList = styled.ul`
+  list-style-type: none;
+  text-align: left;
+  margin-top: 28px;
+`
+
+const WebConferencesListItem = styled.li`
+  margin-bottom: 8px;
+`
+
+const WebConferenceLink = styled.a`
+  text-decoration: none;
+  color: ${props => props.theme.color('neutral', 'bright')};
 `
 
-const Footer = () => (
-  <Container>
-    <p>Footer!</p>
-  </Container>
+const SocialList = styled.ul`
+  list-style-type: none;
+  margin-top: 28px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+`
+
+const SocialListItem = styled.li`
+  width: 34px;
+  height: 34px;
+  margin-right: 16px;
+`
+
+const PastEventItem = props => (
+  <WebConferencesListItem>
+    <WebConferenceLink href={props.path} target="_blank">{props.name}</WebConferenceLink>
+  </WebConferencesListItem>
+)
+
+const SocialItem = props => (
+  <SocialListItem>
+    <a href={props.path}>
+      <img src={socialIcons[props.name]} alt={props.name} />
+    </a>
+  </SocialListItem>
 )
 
+const Footer = ({ pastEvents, social }) => {
+  const pastEventsList = pastEvents.map(pastEvent => (
```

Comment:
I think we should move this to an isolated function outside the `render` function, receiving the `pastEvents` array